virttest/utils_net.py

```diff
@@ -3054,7 +3054,8 @@ def get_ip_address_by_interface(ifname, ip_ver="ipv4"):
         ver = netifaces.AF_INET
     try:
         addr = netifaces.ifaddresses(ifname).get(ver)
-        return [a['addr'] for a in addr if not a['addr'].startswith('fe80')][0]
+        if addr is not None:
+            return [a['addr'] for a in addr if not a['addr'].startswith('fe80')][0]
```

Comment:
This method is to get the ip address and error explain the same.
with this fix, this function won't return anything if it does not find IP